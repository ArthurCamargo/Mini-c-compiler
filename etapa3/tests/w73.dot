digraph G {
n0;
n0 -> n1;
n1;
n1 -> n2;
n1 -> n3;
n2;
n2 -> n4;
n2 -> n5;
n4;
n4 [label="c"];
n5;
n5 [label="5"];
n2 [label="[]"];
n3;
n3 -> n6;
n3 -> n7;
n6;
n6 [label="a"];
n7;
n7 -> n8;
n7 -> n9;
n8;
n8 -> n10;
n8 -> n11;
n10;
n10 [label="2"];
n11;
n11 [label="c"];
n8 [label="*"];
n9;
n9 [label="k"];
n7 [label="+"];
n3 [label="[]"];
n1 [label="="];
n0 [label="f"];

}