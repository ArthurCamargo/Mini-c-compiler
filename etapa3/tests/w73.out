0x5578a3fea350
0x5578a3fea350, 0x5578a3fea740
0x5578a3fea740
0x5578a3fea740, 0x5578a3fea460
0x5578a3fea740, 0x5578a3fea6f0
0x5578a3fea460
0x5578a3fea460, 0x5578a3fea3c0
0x5578a3fea460, 0x5578a3fea410
0x5578a3fea3c0
0x5578a3fea3c0 [label="c"];
0x5578a3fea410
0x5578a3fea410 [label="5"];
0x5578a3fea460 [label="[]"];
0x5578a3fea6f0
0x5578a3fea6f0, 0x5578a3fea4d0
0x5578a3fea6f0, 0x5578a3fea6a0
0x5578a3fea4d0
0x5578a3fea4d0 [label="a"];
0x5578a3fea6a0
0x5578a3fea6a0, 0x5578a3fea5e0
0x5578a3fea6a0, 0x5578a3fea650
0x5578a3fea5e0
0x5578a3fea5e0, 0x5578a3fea520
0x5578a3fea5e0, 0x5578a3fea590
0x5578a3fea520
0x5578a3fea520 [label="2"];
0x5578a3fea590
0x5578a3fea590 [label="c"];
0x5578a3fea5e0 [label="*"];
0x5578a3fea650
0x5578a3fea650 [label="k"];
0x5578a3fea6a0 [label="+"];
0x5578a3fea6f0 [label="[]"];
0x5578a3fea740 [label="="];
0x5578a3fea350 [label="f"];
